linkerd/http/retry/src/lib.rs

```diff
@@ -223,6 +231,8 @@ where
         };
 
         let params = policy.param();
+        let labels = self.extract.extract_param(&req);
```

Comment:
Perhaps worth a TODO that we'd like to avoid knowing about labels here in the future.

Reply:
added in 1c03f91 :hammer: